src/main/java/com/beerjournal/breweriana/item/rating/RatingDto.java

```diff
@@ -0,0 +1,46 @@
+package com.beerjournal.breweriana.item.rating;
+
+import com.beerjournal.breweriana.item.rating.persistence.Rating;
+import lombok.Builder;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+import static com.beerjournal.breweriana.utils.Converters.toObjectId;
+import static com.beerjournal.breweriana.utils.Converters.toStringId;
+import static lombok.AccessLevel.PRIVATE;
+
+@Data
+@Builder
+@RequiredArgsConstructor(access = PRIVATE)
+public class RatingDto {
+
+    private final String id;
+    @NotEmpty
+    private final String itemId;
+    @NotEmpty
+    private final String raterId;
+    @Min(1)
+    @Max(5)
+    private final int value;
+
+    public static RatingDto of(Rating rating) {
+        return RatingDto.builder()
+                .id(rating.getId().toHexString())
```

Comment:
So I will change all or most of them in this pr - not to make overhead with new task.